cmd/main.go

```diff
@@ -1,14 +1,75 @@
 package main
 
 import (
+	"errors"
 	"fmt"
+	"github.com/hashgraph/hedera-sdk-go"
+	hederaClient "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera"
 	"github.com/limechain/hedera-eth-bridge-validator/app/persistence"
+	"github.com/limechain/hedera-eth-bridge-validator/app/persistence/status"
+	consensusmessage "github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/consensus-message"
+	cryptotransfer "github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/crypto-transfer"
 	"github.com/limechain/hedera-eth-bridge-validator/config"
 	"github.com/limechain/hedera-watcher-sdk/server"
+	log "github.com/sirupsen/logrus"
+	"os"
 )
 
 func main() {
+	initLogger()
 	configuration := config.LoadConfig()
-	persistence.RunDb(configuration.Hedera.Validator.Db)
-	server.NewServer().Run(fmt.Sprintf(":%s", configuration.Hedera.Validator.Port))
+	hederaClient := hederaClient.NewHederaClient(configuration.Hedera.MirrorNode.ApiAddress, configuration.Hedera.MirrorNode.ClientAddress)
+	server := server.NewServer()
+
+	db := persistence.RunDb(configuration.Hedera.Validator.Db)
+	statusRepository := status.NewStatusRepository(db)
+
+	failure := addCryptoTransferWatchers(configuration, hederaClient, statusRepository, server)
+	if failure != nil {
+		log.Fatal(failure)
+	}
+
+	failure = addConsensusTopicWatchers(configuration, hederaClient, server)
+	if failure != nil {
+		log.Fatal(failure)
+	}
+
+	server.Run(fmt.Sprintf(":%s", configuration.Hedera.Validator.Port))
+}
+
+func addCryptoTransferWatchers(configuration *config.Config, hederaClient *hederaClient.HederaClient, repository *status.StatusRepository, server *server.HederaWatcherServer) error {
+	if len(configuration.Hedera.Watcher.CryptoTransfer.Accounts) == 0 {
+		log.Warningln("There are no Crypto Transfer Watchers.")
```

Comment:
`There are no Crypto Transfer Watchers.` -> `CryptoTransfer Accounts list is empty. No CryptoTransfer Watchers will be started`